app/views/users/registrations/edit.html.erb

```diff
@@ -0,0 +1,43 @@
+<h2>Edit <%= resource_name.to_s.humanize %></h2>
```

Comment:
Do we need this view? Is this functionality available at all? I'd dump it if not.